ripe/atlas/cousteau/__init__.py

```diff
@@ -165,6 +166,8 @@ def build_url(self):
         for k, v in self.api_filters.items():
             if isinstance(v, datetime):
                 self.api_filters[k] = int(calendar.timegm(v.timetuple()))
+            if isinstance(v, (tuple, list)):
+                self.api_filters[k] = ",".join([str(_) for _ in v])
 
```

Comment:
Can you move the for loop before the second if statement (line 158), so build_url_chunks will see these transformations in case of long id list. And update comment of for loop ;)